### PR TITLE
Improve platform responsiveness for smaller screens

### PR DESCRIPTION
**Pull request by mdshamoon**
## Summary

Improve platform responsiveness for smaller screens

### CODE REVIEW

File: src/containers/List/List.tsx [hunk 1]
```diff
@@ -245,7 +245,7 @@ export const List = ({
     filterItemsQuery,
     {
       variables: filterPayload(),
-      fetchPolicy: 'cache-and-network',
```
Comment (kurund) [inline src/containers/List/List.tsx:248]: Not sure how this is related to responsiveness?

File: src/containers/List/List.tsx [hunk 2]
```diff
@@ -259,7 +259,7 @@ export const List = ({
 
   useEffect(() => {
     refetchCount();
-  }, [filterPayload, searchVal, filters]);
```
Comment (kurund) [inline src/containers/List/List.tsx:262]: Same as above

**Review by kurund — CHANGES_REQUESTED** [review]

File: src/containers/List/List.tsx (hunk 1 above)
Comment (mdshamoon) [reply src/containers/List/List.tsx:248]: When we are opening or closing the sideDrawer this query is getting called again if we have this fetch-policy. I have checked multiple scenarios like adding deleting or editing a list item and there are no issues caused by changing this policy.

File: src/containers/List/List.tsx (hunk 2 above)
Comment (mdshamoon) [reply src/containers/List/List.tsx:262]: refetchcount was getting called even if the filterPayload and filter were not changing while closing or opening the side drawer. Removing these doesnt seem to cause any issue with the existing features.

**Review by kurund — APPROVED** [review]
Looks good and working as expected.

**Pull request merged**